manifests/tar_create.pp

```diff
@@ -9,12 +9,12 @@
 # $foreman_proxy_fqdn::         FQDN of the foreman proxy
 #
 define certs::tar_create(
-  $path               = $title,
-  $foreman_proxy_fqdn = $::certs::foreman_proxy_content::foreman_proxy_fqdn,
+  $path    = $title,
+  $sub_dir = $::certs::foreman_proxy_content::foreman_proxy_fqdn,
```

Comment:
Given we always pass this in I think we should remove the default value here. We're breaking the API anyway by changing the name.